cpp/src/gandiva/tests/projector_test.cc

```diff
@@ -15,13 +15,13 @@
 // specific language governing permissions and limitations
 // under the License.
 
-#include <cmath>
+#include "gandiva/projector.h"
```

Comment:
this reordering looks strange - is this required?

Reply:
yup clang check fails without this